examples/04_react_router/src/App.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  unstable_HistoryRouter as UnstableHistoryRouter,
```

Comment:
Are you okay with this?
```suggestion
  unstable_HistoryRouter as HistoryRouter,
```

Reply:
Sure thing! :ok_hand: